src/seedu/addressbook/data/Tagging.java

```diff
@@ -0,0 +1,35 @@
+package seedu.addressbook.data;
+
+
+import java.util.ArrayList;
+
+public class Tagging {
+    ArrayList<String> tags;
```

Comment:
Recollect the discussion during tutorial on why this is not the right place to maintain the tagging list. Please check and differentiate the 2 diagrams in the activity description for better understanding